src/main/app/src/app/zaken/zaak-afhandelen-dialog/zaak-afhandelen-dialog.component.html

```diff
@@ -0,0 +1,23 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2022 Atos
+  ~ SPDX-License-Identifier: EUPL-1.2+
+  -->
+
+<div mat-dialog-content>
+    <p *ngIf="data.planItem.toelichting" class="readonly" [innerHTML]="data.planItem.toelichting"></p>
+    <mfb-form-field [field]="resultaatFormField"></mfb-form-field>
+    <mfb-form-field [field]="toelichtingFormField"></mfb-form-field>
+    <mfb-form-field [field]="sendMailFormField"></mfb-form-field>
+    <mfb-form-field *ngIf="sendMailFormField.formControl.value" [field]="ontvangerFormField"></mfb-form-field>
+    <p *ngIf="sendMailFormField.formControl.value" class="readonly" [innerHTML]="body"></p>
+</div>
+<div mat-dialog-actions>
+    <button [disabled]="loading || formFieldsInvalid()"
+            mat-button color="primary" (click)="afhandelen()" id="zaakAfhandelen_button">
+        <mat-icon *ngIf="loading">
+            <mat-spinner diameter="20"></mat-spinner>
+        </mat-icon>
```

Comment:
Waarom zit de loading spinner binnen de button als de annuleren-button ook afhangt van `loading`? Zou die spinner ook getoond kunnen worden naast de buttons terwijl ze disabled zijn, of in plaats van de buttons? Is een beetje een smaak-kwestie natuurlijk.

Reply:
Omdat de afhandelen knop een actie uitvoert wat tijd kan kosten, daar krijg je dan een laad indicator te zien. Zodra hij gaat laden mogen ze in de tussentijd niet op annuleren klikken. De annuleren knop hangt dus op dat gebied niet af van de loading, maar heeft enkel een conditie dat hij niet klikbaar mag zijn als de afhandelen knop bezig is.

En gezien de loading vooral is voor de actie afhandelen, wil je dat ook zien op die knop. Als je het er naast zet dan weet je niet waarom hij aan het laden is.